rust/constants/src/test_values.rs

```diff
@@ -0,0 +1,117 @@
+/// Identicons and related pictures, hex encoded vector of png data
+///
+/// Empty 30x30 transparent png image,
+/// used in cases when identicon generation failed or public key does not exist
+pub const EMPTY_PNG: &str = "89504e470d0a1a0a0000000d494844520000001e0000001e08060000003b30aea20000002e49444154789cedcd410100200c0021ed1f7ab6381f8302dc99393f8833e28c3823ce8833e28c3823ce8833fbe20724cf59c50a861d5c0000000049454e44ae426082";
```

Comment:
either `include_str` or `include_bytes` and convert to necessary data in code here and in all the following constants.